treeherder/perf/secretary_tool.py

```diff
@@ -5,11 +5,13 @@
 from django.conf import settings as django_settings
 
 from treeherder.perf.models import BackfillRecord, BackfillReport, PerformanceSettings
+from treeherder.utils import default_serializer
 
 logger = logging.getLogger(__name__)
 
 
 # TODO: update the backfill status using data
+# TODO: consider making this a singleton
 class SecretaryTool:
     """
     Tool used for doing the secretary work in the Performance Sheriff Bot.
```

Comment:
I think a clearer message of what it's doing would be helpful. I don't know what you mean by secretary work :)